README_DOCKERHUB.md

```diff
@@ -4,35 +4,37 @@ Docker images with out-of-the-box [SonarQube LTS](https://www.sonarqube.org) ins
-- `2.4.0`, `latest` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.4.0/Dockerfile), [(v2.4.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.4.0)
+- `2.4.0` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.4.0/Dockerfile), [(v2.4.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.4.0)
 - `2.4.0-full` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.4.0-full/Dockerfile), [(v2.4.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.4.0)
-- `2.3.0`, `latest` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.3.0/Dockerfile), [(v2.3.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.3.0)
+- `2.3.0` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.3.0/Dockerfile), [(v2.3.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.3.0)
 - `2.3.0-full` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.3.0-full/Dockerfile), [(v2.3.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.3.0)
-- `2.2.1`, [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.2.1/Dockerfile), [(v2.2.1)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.2.1)
+- `2.2.1` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.2.1/Dockerfile), [(v2.2.1)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.2.1)
 - `2.2.1-full` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.2.1-full/Dockerfile), [(v2.2.1)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.2.1)
 - `2.2.0` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.2.0/Dockerfile), [(v2.2.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.2.0)
 - `2.2.0-full` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.2.0-full/Dockerfile), [(v2.2.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.2.0)
 - `2.1.0` [Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.1.0/Dockerfile), [(v2.1.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.1.0)
 - `2.1.0-full` ([Dockerfile](https://github.com/mwz/sonar-scala-docker/blob/master/2.1.0-full/Dockerfile)), [(v2.1.0)](https://github.com/mwz/sonar-scala-docker/releases/tag/2.1.0)
 
 ## What's included
-Version | SonarQube | sonar-scala | sonar-scala-extra
---------|-----------|-------------|------------------
-2.5.0 | | 6.4.0 | 1.3.0
-2.5.0-full | 6.7.3 LTS | 6.4.0 | 1.3.0
-2.4.0 | | 6.3.0 | 1.3.0
-2.4.0-full | 6.7.3 LTS | 6.3.0 | 1.3.0
-2.3.0 | | 6.2.0 | 1.3.0
-2.3.0-full | 6.7.3 LTS | 6.2.0 | 1.3.0
-2.2.1 | | 6.1.0 | 1.3.0
-2.2.1-full | 6.7.3 LTS | 6.1.0 | 1.3.0
-2.2.0 | | 6.1.0 | 1.3.0
-2.2.0-full | 6.7.2 LTS | 6.1.0 | 1.3.0
-2.1.0 | | 6.0.0 | 1.3.0
-2.1.0-full | 6.7.1 LTS | 6.0.0 | 1.3.0
-
+Version | SonarQube | PostgreSQL | sonar-scala | sonar-scala-extra
```

Comment:
As mentioned previously, Postgres isn't included in the docker images and we should exclude the `PostgreSQL` column from this table.